.github/workflows/pr-e2e.yml

```diff
@@ -7,8 +7,27 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: skip for markdown only prs
+        id: one
+        run: |
+          current_branch=$(git rev-parse --abbrev-ref HEAD)
+          # diff of branch excluding md
+          testable_changes=$(git diff --name-only "$current_branch" $(git merge-base "$current_branch" origin/master) -- . ':(exclude)*.md')
+          echo "changed files: $testable_changes"
+          # skip if there are only md changes
+          if [ -z "$testable_changes" ]; then
+            echo "skip e2e"
+            echo "::set-output name=skip::skip"
```

Comment:
Nitpick: does this output need a value set here if it is not used?

Reply:
God question, I'll look into that. I was just being safe because I didn't know what would happen to the conditional below if the variable was undefined